### ISSUE TITLE
Feature: CoreWiki as plugin in Microsoft Teams

### ISSUE THREAD

**Issue by csharpfritz** [csharpfritz/CoreWiki#289]
I would like to be able to add Corewiki as a wiki panel in Microsoft Teams.

**Comment by summer600** [comment]
I don't think you can do it via the plugin store because unless we build out multiple tenancy the corewiki instance would be a private one hosted by us, the team owner.
You would also need to connect AzureAD security and preferably do roles through AzureAD as well. You could do without this integration and AzureAD can be a separate feature.

**Comment by csharpfritz** [comment]
Good points... and features that we could build. 



Jeff
